File: 1-Beginner/1074/bee1074.java
// 1074 - Par ou Ímpar
import java.io.IOException;
import java.util.Scanner;

class bee1074 {
    
    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);
        
        int n = scan.nextInt();
        for(; n > 0; n--) {
            int x = scan.nextInt();
            
            if(x > 0) {
                if(x % 2 == 0) System.out.println("EVEN POSITIVE");
                else System.out.println("ODD POSITIVE");
            }
            else if(x< 0) {
                if(x % 2 == 0) System.out.println("EVEN NEGATIVE");
                else System.out.println("ODD NEGATIVE");
            }
            else System.out.println("NULL");
        }
        
        scan.close();
    }
}

File: 1-Beginner/1074/bee1074.c
// 1074 - Par ou Ímpar
#include <stdio.h>

int main(void) {
    
    int n; scanf("%d", &n);
    for(; n > 0; n--) {
        int x; scanf("%d", &x);
        
        if(x > 0) {
                if(x % 2 == 0) printf("EVEN POSITIVE\n");
                else printf("ODD POSITIVE\n");
            }
            else if(x< 0) {
                if(x % 2 == 0) printf("EVEN NEGATIVE\n");
                else printf("ODD NEGATIVE\n");
            }
            else printf("NULL\n");
    }
    
    return 0;
}